certificate-manager/src/main/java/io/strimzi/certs/OpenSslCertManager.java

```diff
@@ -176,45 +144,26 @@ public void deleteFromTrustStore(List<String> aliases, File trustStoreFile, Stri
 
     @Override
     public void renewSelfSignedCert(File keyFile, File certFile, Subject sbj, int days) throws IOException {
-        // See https://serverfault.com/questions/306345/certification-authority-root-certificate-expiry-and-renewal
+        // See https://serverfault.com/a/501513
 
         //openssl req -new -key root.key -out newcsr.csr
-        File csrFile = File.createTempFile("renewal", ".csr");
+        File csrFile = Files.createTempFile("renewal", ".csr").toFile();
```

Comment:
why this change? What's the actual difference between `createTempFile` in `File` vs `Files` (other than the need to call one more `toFile` on the second one?)

Reply:
The javadoc for `File.createTempFile` includes:

> The Files.createTempFile method provides an alternative method to create an
> empty file in the temporary-file directory. Files created by that method
> may have more restrictive access permissions to files created by this
> method and so may be more suited to security-sensitive applications.

IDK the details, and maybe in a container there's no real difference, but since we're messing around with keys in this code, I felt this was worth changing, just to be on the safe side.